site/docs/v1/tech/apis/consents.adoc

```diff
@@ -166,6 +167,63 @@ This API does not return a JSON response body.
 
 include::docs/v1/tech/apis/_standard-delete-response-codes.adoc[]
 
+== Search for Consents
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Consents and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Consents by name
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/consent/search``?name=\{name\}``#
+--
+
+=== Request Parameters
+
+:parameter_prefix:
+include::docs/v1/tech/apis/_consent-search-request-parameters.adoc[]
+:parameter_prefix!:
+
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Consents
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/consent/search#
+--
+
+When calling the API using a `POST` request you will send the search criteria in a JSON request body.
+
+=== Request Body
+
+:parameter_prefix: search.
+include::docs/v1/tech/apis/_consent-search-request-parameters.adoc[]
+:parameter_prefix!:
+
+=== Response
+
+The response for this API contains the Consents matching the search criteria in paginated format and the total number of results matching the search criteria.
+
+:never_missing:
+:never_search_error:
+include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
+:never_missing!:
+:never_search_error!:
+
+:include_total:
+include::docs/v1/tech/apis/_consents-response-body.adoc[]
+:include_total!:
+
+
 == Grant a User Consent
 
```

Comment:
@mooreds the primary issue was about adding search to admin UI pages and mentioned `Consents`. I think `UserConsents` appear on the manage user page.

We could add a `UserConsent`s search, but it would probably be API only to start with.

Reply:
Okay, I filed an issue here: https://github.com/FusionAuth/fusionauth-issues/issues/2099 . Let's leave it be for now.